packages/sdk-core/src/BacktraceCoreClient.ts

```diff
@@ -71,11 +77,15 @@ export abstract class BacktraceCoreClient {
     }
 
     public get metrics(): BacktraceMetrics | undefined {
-        return this._metrics;
+        return this._modules.get(BacktraceMetrics);
```

Comment:
did you merge your code from dev to this branch? You already landed this change

Reply:
due to merging previous stuff, I had to rebase.